FacuReartes/react-laba-final-project-team-2
# [SPT2-47] Forgot and Reset Password - Functionality

# Description
Email Validation: The entered email address should be validated to ensure it is in the correct format.

Password Reset Process: Once the user submits their email address, an email with a password reset link should be sent to that email address. The email should include instructions on how to reset the password.

## Changes
- Feedback window (Generic Popup)
- Forgot password implementation to backend endpoint
- Reset passoword implementation to backend endpoint

### Jira

- [SPT2-106] - Feedback window
- [SPT2-47] - Forgot and Reset Password - Functionality

### How to Test

1. Navigate to `/forgot-password` and fill the email field with registed email.
2. Click in `Reset password` button
3. Verify in your email account the with subject `Administration Panel`.
4. Click in the link in email to navigate to `/reset-password` page.
5. Insert new `password` and `confirmation password`
6. Click in `Reset password` button

### Screenshots 
![image](https://github.com/user-attachments/assets/e2bca4d7-dc42-4d62-8c60-e0c947caa890)

### Checklist



[SPT2-106]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-106?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[SPT2-47]: https://solvd-final-project-team-2.atlassian.net/browse/SPT2-47?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
